pom.xml

```diff
@@ -91,6 +91,7 @@
     <gson.version>2.2</gson.version>
     <guava.version>15.0</guava.version>
     <jetty.version>9.2.15.v20160210</jetty.version>
+    <hadoop.hdfs.version>2.6.0</hadoop.hdfs.version>
```

Comment:
how about using `hadoop.version` ? Since in spark-dependencies/pom.xml, we also define hadoop.verison. We can reuse the same property name. Otherwise user need to specify both `hadoop.version` and `hadoop.hdfs.version` when building with custom hadoop.